Deeploy/Targets/GAP9/Templates/NE16GEMMTemplate.py

```
@@ -0,0 +1,242 @@
+        input_signed = A._type.referencedType.typeMin < 0
+        output_bits = data_out._type.referencedType.typeWidth
+        output_signed = data_out._type.referencedType.typeMin < 0
+
+        operatorRepresentation['input_signed'] = input_signed
+        operatorRepresentation['output_bits'] = output_bits
+        operatorRepresentation['quant_bits'] = 2 if output_bits == 32 else 0
+        operatorRepresentation['quant_norect'] = 1 if (output_bits == 32 or output_signed) else 0
+
+        # Weight packing and signed bias compensation
+        w_int8 = B.values.astype(np.int8)  # [Ko, Ki], still int8 at this point
+        Ko, Ki = w_int8.shape
```

Ruff (0.15.10)

[warning] Unpacked variable `Ki` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)